src/style.css

```diff
@@ -0,0 +1,113 @@
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.box {
+  display: flex;
+  height: 300px;
+  min-width: 300px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+  color: #fff;
+}
+
+.box-1 {
+  background-color: rgb(100%, 0%, 0%);
+  order: 1;
+}
+
+.box-2 {
+  background-color: rgb(80%, 0%, 0%);
+  order: 2;
+}
+
+.box-3 {
+  background-color: rgb(60%, 0%, 0%);
+  order: 3;
+}
+
+.box-4 {
+  background-color: rgb(40%, 0%, 0%);
+  order: 4;
+}
+
+.box-5 {
+  background-color: rgb(20%, 0%, 0%);
+  order: 5;
+}
+
+.box-6 {
+  background-color: rgb(0%, 0%, 0%);
+  order: 6;
+}
+
+@media (min-width: 0) and (max-width: 599px) {
+  .box {
+    flex-basis: 100%;
+  }
+}
```

Comment:
If you want to use a mobile-first approach, style your blocks with 100% width by default. Don't add too many media queries